mobile-browser-based-version/package.json

```diff
@@ -7,7 +7,8 @@
     "prod": "vue-cli-service serve --mode production",
     "build": "vue-cli-service build",
     "lint": "vue-cli-service lint",
-    "test": "env TS_NODE_COMPILER_OPTIONS='{\"module\": \"commonjs\" }' mocha --exit -r ts-node/register --file tests/setup.ts 'tests/**/*.ts'",
+    "blam": "false; true",
```

Comment:
what is blam for?

Reply:
hoo, good catch, it was a debug line; removing it